radixdlt/src/main/java/com/radixdlt/consensus/PendingVotes.java

```diff
@@ -31,48 +31,35 @@
  * Manages pending votes for various vertices
  */
 public final class PendingVotes {
-	private final QuorumRequirements quorumRequirements;
 	private final HashMap<Hash, ECDSASignatures> pendingVotes = new HashMap<>();
 
-	@Inject
-	public PendingVotes(QuorumRequirements quorumRequirements) {
-		this.quorumRequirements = Objects.requireNonNull(quorumRequirements);
-	}
-
 	/**
 	 * Inserts a vote for a given vertex, attempting to form a quorum certificate for that vertex.
 	 *
-	 * The vote will only be inserted if permitted by the installed {@link QuorumRequirements}.
-	 * Similarly, a QC will only be formed if permitted by the installed {@link QuorumRequirements}.
+	 * The vote will only be inserted if permitted by the {@link ValidatorSet}.

```

Comment:
This isn't technically true, a vote will be inserted into `signatures` even if `validatorSet` doesn't like it since there is no explicit check. The only check `validatorSet` is used for is whether a QC has formed.